diagnostic.md

```diff
@@ -5,74 +5,88 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```bash
-// your response here
+to keep the data safe(er) and more secure and to access it for separation of concerns from the front end.
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```bash
-// your response here
+the model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```bash
-// your response here
+the controller
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```bash
-// your response here
+they only copy back to the user what the controller gives them
 ```
```

Comment:
The important distinction here is that we use json instead, because SPA's are a better standard.